### PR TITLE
feat(login-form): Login Form component

### PR DESCRIPTION
**Pull request by mrviniciux**
## Description

image

This PR accomplish initial tests and basic skeleton structure to login form. I decided to not include the entire /login page because the PR became huge just by creating the basic component. So this is not the final version of /login page.

Also storybook isn't included due to a problem with storybook internal config. I opened an issue at boilerplate-next:

https://github.com/grupo-sbf/boilerplate-next/issues/28

## What was done in this PR?

- [x] Task already done
- [ ] Task to be done

### CODE REVIEW

**Comment by mrviniciux** [comment]
Texts and element positions are based on figma:

https://www.figma.com/file/VCJ5CAdMZcjxhyjRdlYIlD/Melhorias-de-Heur%C3%ADstica?node-id=8%3A8567

File: src/common/components/LoginForm/LoginForm.constants.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  LABEL_PASSWORD: 'Senha',
+  RECOVER_PASSWORD: 'Recuperar minha senha',
+  SIGN_IN: 'Entrar',
+};
+
+export const ERROR = {
+  MAIL: 'Usuário/E-mail incorreto',
+  INVALID_MAIL: 'E-mail inválido',
+  PASSWORD: 'Senha incorreta',
+};
```
Comment (bin-fmmoraes) [inline src/common/components/LoginForm/LoginForm.constants.ts:19]: I like this ideia of having a `.constant` file, we should addopt for other projects

File: src/common/components/GlobalStyle/GlobalStyle.tsx [hunk 2]
```diff
@@ -6,10 +6,6 @@ export const GlobalStyle = createGlobalStyle`
     padding: 0;
     box-sizing: border-box;
   }
-
-  html {
-    font-size: calc(14em * .0625);
-  }
```
Comment (bin-fmmoraes) [inline src/common/components/GlobalStyle/GlobalStyle.tsx:12]: What is this?

File: src/common/components/GlobalStyle/GlobalStyle.tsx [hunk 3]
```diff
@@ -6,10 +6,6 @@ export const GlobalStyle = createGlobalStyle`
     padding: 0;
     box-sizing: border-box;
   }
-
-  html {
-    font-size: calc(14em * .0625);
```
Comment (bin-fmmoraes) [inline src/common/components/GlobalStyle/GlobalStyle.tsx:11]: I don't think we should change this.
[Check Fairplay documentation](https://fairplay-design-system.web.app/doc/getting-started/MeasureUnits)

File: src/common/components/LoginForm/icons/Visibility.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import Img from '@/common/components/Img';
+
+const Visibility = () => (
```
Comment (bin-fmmoraes) [inline src/common/components/LoginForm/icons/Visibility.tsx:3]: There's a Visibility icon on Fairplay. Check their [icons documentation](https://fairplay-design-system.web.app/doc/Icones)

**Review by bin-fmmoraes — APPROVED** [review]
I see you are following our guidelines and using other projects as example (Nike).
Keep going! :rocket:

**Pull request merged**